calcom/cal.com
# Removed v2 pages directory, moved all pages to root and deleted old v1 pages and middleware redirect.

## What does this PR do?

Removed v2 pages directory, moved all pages to root and deleted old v1 pages and middleware redirect.

In this PR there have been done **zero** code changes to the pages, the diffs you see happened because the filenames between v1 and v2 overlap, I only moved the pages back to the root directory.

**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

This could in theory impact the whole app, therefore it's important to click around the app and ensure no URL's return a 404 (did test this myself as well). Main urls impacted are:
* /settings/...
* /apps/...
* /teams
* /auth/login
* /availability/...
* /event-types/...
* /bookings/[status]
* /success
